server/test/src/main/antlr4/org/eclipse/usecase/UseCasePreprocessor.g4

```diff
@@ -167,7 +166,7 @@ DOT : '.';
 
 NUMBERLITERAL : [\-+0-9.,]+;
 STRINGLITERAL : ['"] .*? ['"\n];
-IDENTIFIER : [a-zA-Z0-9:]+ ([-_]+ [a-zA-Z0-9:]+)*;
+IDENTIFIER : [a-zA-Z0-9'":]+ ([-_]+ [a-zA-Z0-9'":]+)*;
```

Comment:
Could you provide an example where this is a valid identifier?

Reply:
It's not a valid identifier, but here what we can do https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/5df1d021ad79ee139d955b4b06c626a28e4af8e1/server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java#L91